pkg/storaged/block/format-dialog.jsx

```diff
@@ -180,6 +180,7 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
     add_fsys("ext4", { value: "ext4", title: "EXT4" });
     add_fsys("vfat", { value: "vfat", title: "VFAT" });
     add_fsys("ntfs", { value: "ntfs", title: "NTFS" });
+    add_fsys("btrfs", { value: "btrfs", title: "BTRFS" });
```

Comment:
Oho!

Reply:
Yes, woops this slipped in by accident. Working on it in a new branch